tracekit.js

```diff
@@ -233,11 +233,10 @@ TraceKit.report = (function reportModuleWrapper() {
      */
     function processLastException() {
         var _lastExceptionStack = lastExceptionStack,
-            _lastArgs = lastArgs;
-        lastArgs = null;
+            _lastException = lastException;
         lastExceptionStack = null;
         lastException = null;
-        notifyHandlers.apply(null, [_lastExceptionStack, false].concat(_lastArgs));
+        notifyHandlers(_lastExceptionStack, false, _lastException);
```

Comment:
So there was an undocumented feature: if you call `TraceKit.report(e, ...rest);`, it seems `...rest` arguments were passed all the way to subscribers. I could not find it documented anywhere, and had to remove that (in f02628afa656ada15bbd59ef6e4f43fdee6a0c94)